receiver/splunkhecreceiver/README.md

```diff
@@ -30,10 +30,10 @@ The following settings are optional:
     * `key_file`: Specifies the key file to use for TLS connection. Note: Both
       `key_file` and `cert_file` are required for TLS connection.
 * `path` (default = '/*'): The path to listen on, as a glob expression.
-* `source_key` (default = 'com.splunk.source'): Specifies the source field to a specific unified model attribute.
-* `sourcetype_key` (default = 'com.splunk.sourcetype'): Specifies the sourcetype field to a specific unified model attribute.
-* `index_key` (default = 'com.splunk.index'): Specifies the index field to a specific unified model attribute.
-* `host_key` (default = 'host.name'): Specifies the host field to a specific unified model attribute.
+* `hec_to_otel_attrs/source` (default = 'com.splunk.source'): Specifies the mapping of the source field to a specific unified model attribute.
+* `hec_to_otel_attrs/sourcetype` (default = 'com.splunk.sourcetype'): Specifies the mapping of the sourcetype field to a specific unified model attribute.
+* `hec_to_otel_attrs/index` (default = 'com.splunk.index'): Specifies the mapping of the  index field to a specific unified model attribute.
+* `hec_to_otel_attrs/host` (default = 'host.name'): Specifies the mapping of the host field to a specific unified model attribute.
```

Comment:
Please add a line to CHANGELOG saying that this is a breaking change for the receiver config interface

Reply:
I don't think it should be considered a breaking change, since we apply all the current defaults.

Reply:
Technically I think it is a breaking change in config interface. If user already uses any of these fields: `source_key`, `sourcetype_key`, `index_key`, `host_key`, collector will fail to start

Reply:
ah true. Will add.

Reply:
added to CHANGELOG